wicker/core/datasets.py

```diff
@@ -175,7 +335,9 @@ def _get_dataset_size(self):
         arrow_table = self.arrow_table()
 
         buckets_keys = set()
-        worker = ShuffleWorker(storage=self._storage)
```

Comment:
Just a reminder that we'll update this line in https://github.com/woven-planet/wicker/pull/61 so depending on the order in which we merge the PRs there will be some updates here.